guacamole/src/main/frontend/src/translations/ca.json

```diff
@@ -163,6 +183,80 @@
+        "ACTION_IMPORT_CONNECTIONS" : "Importa connexions",
+
+        "DIALOG_HEADER_ERROR"   : "@:APP.DIALOG_HEADER_ERROR",
+        "DIALOG_HEADER_SUCCESS" : "Fet",
+
+        "ERROR_AMBIGUOUS_CSV_HEADER"         : "La capçalera \"{HEADER}\" del CSV és ambigua, pot ser o bé un atribut de connexió o un paràmetre",
+        "ERROR_AMBIGUOUS_PARENT_GROUP"       : "No es pot especificar el grup i el parentIdentifier alhora",
+        "ERROR_ARRAY_REQUIRED"               : "El fitxer ha de contenir un llistat de connexions",
+        "ERROR_DETECTED_INVALID_TYPE"        : "Tipus de fitxer no suportat. Assegureu-vos que és un fitxer CSV, JSON o YAML vàlid.",
+        "ERROR_DUPLICATE_CONNECTION_IN_FILE" : "Nom de connexió \"{NAME}\" duplicat a \"{PATH}\" al fitxer d'importació",
+        "ERROR_DUPLICATE_CSV_HEADER"         : "Capçalera duplicada al CSV: {HEADER}",
+        "ERROR_EMPTY_FILE"                   : "El fitxer està buit",
+        "ERROR_INVALID_CSV_HEADER"           : "La capçalera \"{HEADER}\" del CSV no és vàlida, no és ni un atribut ni un paràmetre",
+        "ERROR_INVALID_MIME_TYPE"            : "Tipus de fitxer no suportat: \"{TYPE}\"",
+        "ERROR_INVALID_GROUP"                : "No s'ha trobat cap grup que coincideixi amb \"{GROUP}\"",
+        "ERROR_INVALID_GROUP_IDENTIFIER"     : "No s'ha trobat cap grup de connexió amb l'identificador \"{IDENTIFIER}\"",
+        "ERROR_INVALID_GROUP_TYPE"           : "El grup no és vàlid - ha de ser una cadena de text.",
+        "ERROR_INVALID_PROTOCOL"             : "El protocol no és vàlid \"{PROTOCOL}\"",
+        "ERROR_INVALID_USER_GROUPS_TYPE"     : "El grup d'usuari no és vàlid - ha de ser un array d'identificadors de grup d'usuari.",
+        "ERROR_INVALID_USERS_TYPE"           : "Usuaris no vàlids - ha de ser un array d'identificadors d'usuari.",
+        "ERROR_NO_FILE_SUPPLIED"             : "Seleccioneu un fitxer per importar",
+        "ERROR_PARSE_FAILURE_CSV"            : "Assegureu-vos que heu pujat un fitxer CSV vàlid. L'anàlisi ha fallat amb el següent error \"{ERROR}\". ",
+        "ERROR_PARSE_FAILURE_JSON"           : "Assegureu-vos que heu pujat un fitxer JSON vàlid. L'anàlisi ha fallat amb el següent error \"{ERROR}\". ",
+        "ERROR_PARSE_FAILURE_YAML"           : "Assegureu-vos que heu pujat un fitxer YAML vàlid. L'anàlisi ha fallat amb el següent error \"{ERROR}\". ",
+        "ERROR_REJECT_UPDATE_CONNECTION"     : "La connexió \"{NAME}\" ja existeix a \"{PATH}\"",
+        "ERROR_REQUIRED_NAME_CONNECTION"     : "El nom de connexió és obligatori",
+        "ERROR_REQUIRED_PROTOCOL_CONNECTION" : "El protocol de connexió és obligatori",
+        "ERROR_REQUIRED_NAME_FILE"           : "No s'ha trobat el nom de la connexió al fitxer",
+        "ERROR_REQUIRED_PROTOCOL_FILE"       : "No s'ha trobat el protocol de connexió al fitxer",
+
+        "FIELD_PLACEHOLDER_FILTER" : "@:APP.FIELD_PLACEHOLDER_FILTER",
```

Comment:
This should be below the `FIELD_HEADER` items to maintain alphabetical order.

Reply:
Reorganized